tests/test_json_schema_utils.py

```diff
@@ -8,11 +8,35 @@
 
 
 def compare_dicts(a: dict, b: dict):
-    assert json.dumps(a, sort_keys=True, indent=2) == json.dumps(b, sort_keys=True, indent=2)
+    a = sort_dict(a)
+    b = sort_dict(b)
+    assert json.dumps(a, indent=2) == json.dumps(b, indent=2)
 
 
 def compare_dicts_2(a: dict, b: dict):
-    assert json.dumps(a, sort_keys=True) == json.dumps(b, sort_keys=True)
+    a = sort_dict(a)
+    b = sort_dict(b)
+    assert json.dumps(a) == json.dumps(b)
```

Comment:
If a dict has a list or other dicts as keys, then `json.dumps(d, sort_keys=True)` does not sort recursively.